<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastre_option_form</class>
 <widget class="QDialog" name="cadastre_option_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastre</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>523</width>
        <height>805</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Interface QGIS</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Vous pouvez choisir d'appliquer une interface simplifiée de QGIS pour consulter le cadastre ou de revenir à l'interface par défaut</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="btInterfaceCadastre">
              <property name="text">
               <string>Interface Cadastre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btInterfaceQgis">
              <property name="text">
               <string>Interface QGIS</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Mot-clés pour trouver les fichiers MAJIC</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Cette configuration permet de définir comment le plugin trouve les fichiers dans le répertoire MAJIC, en fonction de chaque type de fichier. Un simple mot ou une expression régulière peuvent être utilisés.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="5" column="2">
             <widget class="QLineEdit" name="inMajicProp">
              <property name="text">
               <string>PROP</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>PDL</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>BATI</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="inMajicBati">
              <property name="text">
               <string>BATI</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="inMajicPdl">
              <property name="text">
               <string>PDL</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>PROP</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="inMajicNbati">
              <property name="text">
               <string>NBAT</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="inMajicLotlocal">
              <property name="text">
               <string>LLOC|D166</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>NBATI</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>LOTLOCAL</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLineEdit" name="inMajicTopo">
              <property name="text">
               <string>TOPO</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TOPO</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Modèle de composition pour l'export de la vue cartographique</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="inComposerTemplateFile"/>
            </item>
            <item>
             <widget class="QToolButton" name="btComposerTemplateFile">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Répertoire temporaire</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="inTempDir"/>
            </item>
            <item>
             <widget class="QToolButton" name="btTempDir">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Performances</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Nombre de lignes MAJIC insérées par lot</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="inMaxInsertRows">
              <property name="minimum">
               <number>1000</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="value">
               <number>50000</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Spatialite - stockage temporaire</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="inSpatialiteTempStore">
              <item>
               <property name="text">
                <string>MEMORY</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DEFAULT</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>btInterfaceCadastre</tabstop>
  <tabstop>btInterfaceQgis</tabstop>
  <tabstop>inMajicBati</tabstop>
  <tabstop>inMajicLotlocal</tabstop>
  <tabstop>inMajicNbati</tabstop>
  <tabstop>inMajicPdl</tabstop>
  <tabstop>inMajicProp</tabstop>
  <tabstop>inMajicTopo</tabstop>
  <tabstop>inComposerTemplateFile</tabstop>
  <tabstop>btComposerTemplateFile</tabstop>
  <tabstop>inTempDir</tabstop>
  <tabstop>btTempDir</tabstop>
  <tabstop>inMaxInsertRows</tabstop>
  <tabstop>inSpatialiteTempStore</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
